interface/weno/drugDataInsert.php

```diff
@@ -18,23 +18,28 @@
     csrfNotVerified();
 }
 
```

Comment:
What kind of user should have access to this script (figuring out best acl_check aco that should be used here).

Also, would it make sense to truncate the table (ie. remove the data in case it is there) in case a user is upgrading the data in the future?

Reply:
Truncating data for future data updates is a good idea and one that is in the works. We have been discussing how to implement and update system.

Reply:
To your first question, I would stick with admin level or super admin.

Reply:
Then would add the following code here:
if (!acl_check('admin', 'super')) {
    exit;
}

Reply:
Above post still needs to be addressed